### PR TITLE
Close #203: add flag to skip  task 'install VS Code repo'

### LINKED ISSUE #203: add possibility to skip the task `install VS Code repo (apt/yum)`
implement flag `visual_studio_code_add_repo` with a default of true, which could be used to disable adding the repo 

this feature request was discussed here: https://github.com/gantsign/ansible-role-visual-studio-code/pull/200#issuecomment-1010463138 as an alternativ

### PR DESCRIPTION
**Pull request by arolfes**
introduce a new flag `visual_studio_code_skip_add_repo`

if set to yes, the task 'install VS Code repo (apt/yum/dnf/zypper)' will be skipped
if set to no, the repo will be added, this is the default

### CODE REVIEW

File: README.md
Comment (freemanjp) [inline README.md:62]: Please update this to use `yes`/`no` as below.

**Review by freemanjp — CHANGES_REQUESTED** [review]

File: README.md
Comment (arolfes) [reply README.md]: i added the change into a separate commit.
i can squash it if you like

**Review by freemanjp — APPROVED** [review]

File: README.md
Comment (freemanjp) [reply README.md]: Sorry, I must have started the review between these changes, all good now.

**Comment by freemanjp** [comment]
@arolfes thanks for this pull request. I'll wait until your other change is merged as well before releasing the changes.

**Pull request merged**
